server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
@maheshchnt you shouldn't need to duplicate this file here, you should be able to share with the char driver.